libpod/container.go

```diff
@@ -198,14 +269,23 @@ func (c *Container) PID() (int, error) {
 func (c *Container) MountPoint() (string, error) {
 	c.lock.Lock()
 	defer c.lock.Unlock()
-
 	if err := c.runtime.state.UpdateContainer(c); err != nil {
 		return "", errors.Wrapf(err, "error updating container %s state", c.ID())
 	}
-
 	return c.state.Mountpoint, nil
 }
 
+// GetConfig returns the container config
```

Comment:
I think we have a Config() that gets the config already

Reply:
yeah you added it I believe but it doesnt do error handling and our irc conversation was that we needed to handle that.

Reply:
I think this one doesn't need error handling because the state stuff might change but this should never be modified. Not 100% on that though. Will take a look later and update on whether it is necessary.

Reply:
Checked the code, am now certain that we don't need the syncContainer() call or locking here. The only parts of the container that can be updated are in container.state, container.config should never change (the database will never change that table aside from removing deleted containers, and the sync call does not touch it at all). So we can safely use container.Config() instead, which I think simplifies some things because you don't need to worry about error handling as much.

Reply:
Delete and using Config() now